test/unit/FileAppendTransaction.js

```diff
@@ -88,4 +88,52 @@ describe("FileAppendTransaction", function () {
         expect(body.fileAppend.contents.length).to.be.equal(1000);
         expect(body.fileAppend.contents[0]).to.be.equal(51);
     });
+
+    it("setChunkInterval()", function () {
+        const spenderAccountId1 = new AccountId(7);
+        const fileId = new FileId(8);
+        const nodeAccountId = new AccountId(10, 11, 12);
+        const timestamp1 = new Timestamp(14, 15);
+        const chunkInterval = 200;
+
+        let transaction = new FileAppendTransaction()
+            .setTransactionId(
+                TransactionId.withValidStart(spenderAccountId1, timestamp1),
+            )
+            .setNodeAccountIds([nodeAccountId])
+            .setFileId(fileId)
+            .setChunkSize(1000)
+            .setChunkInterval(chunkInterval)
+            .setContents("1".repeat(1000) + "2".repeat(1000) + "3".repeat(1000))
+            .freeze();
+
+        expect(transaction._transactionIds.list.length).to.be.equal(3);
+
+        for (let i = 0; i < 3; i++) {
```

Comment:
Can we have the content separated in another variable/const:

`const contents = "1".repeat(1000) + "2".repeat(1000) + "3".repeat(1000)
const requiredChunks = contents.length/chunkSize` 

This way we can have `for(let I=0; I< requiredChunks; I++` and then inside the tests I don't like the `if`s. Can't we just check the difference between current transaction and previous transaction to equal chunkInterval?

Reply:
Adressed